public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -101,7 +105,7 @@ export class CloudWatchDatasource
 
     const dataQueryResponses: Array<Observable<DataQueryResponse>> = [];
     if (logQueries.length) {
-      dataQueryResponses.push(this.logsQueryRunner.handleLogQueries(logQueries, options, super.query.bind(this)));
```

Comment:
when looking at the CloudwatchLogsQueryRunner it took a bit of time just to understand what calls what so I wanted to emphasize that this is a public function. It looks a little funny now in comparison to the other runners but hopefully it's helpful?